worlds/witness/Options.py

```diff
@@ -54,27 +92,33 @@ class TrapPercentage(Range):
     default = 20
 
 
-the_witness_options: Dict[str, Option] = {
+the_witness_options: Dict[str, type] = {
     # "hard_mode": HardMode,
-    # "unlock_symbols": UnlockSymbols,
     "disable_non_randomized_puzzles": DisableNonRandomizedPuzzles,
     "shuffle_discarded_panels": ShuffleDiscardedPanels,
     "shuffle_vault_boxes": ShuffleVaultBoxes,
     "shuffle_uncommon": ShuffleUncommonLocations,
     "shuffle_hard": ShuffleHardLocations,
-    "challenge_victory": ChallengeVictoryCondition,
-    "trap_percentage": TrapPercentage
+    "victory_condition": VictoryCondition,
+    "trap_percentage": TrapPercentage,
+    "early_secret_area": EarlySecretArea,
+    # "shuffle_symbols": ShuffleSymbols,
+    # "shuffle_doors": ShuffleDoors,
+    "mountain_lasers": MountainLasers,
+    "challenge_lasers": ChallengeLasers,
 }
 
 
 def is_option_enabled(world: MultiWorld, player: int, name: str) -> bool:
     return get_option_value(world, player, name) > 0
 
 
-def get_option_value(world: MultiWorld, player: int, name: str) -> int:
+def get_option_value(world: MultiWorld, player: int, name: str) -> any:

```

Comment:
Does this truly return anything or is it a `Union[bool, int]` or something else? I'd just want this to be as specific as possible.

Reply:
also any is not any type, it's a chained or; used like any(my_condition1, my_condition_2, my_list1) -> bool

Reply:
It should be Union[bool, int] atm as we only use integer or boolean options

but if we add more complex options in the future Any might be better

Reply:
I'll change it to Union[bool,int], so far that's all it can be I think :)